CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 4.15.x
 
+* Fix possible crash when recording reports and breadcrumbs containing values
```

Comment:
Would be good to include a link to the PR so anybody looking at the changelog/release notes can consult the changeset